### PR TITLE
Landing page code refactored

### PR DESCRIPTION
**Pull request by DevRaahul**
1. Removed unwanted CSS
2. Refactored landing page
image
3. Added installation page.
image
4.  Will update installation link once done with merging

### CODE REVIEW

**Review by abutaher11 — APPROVED** [review]
Looks good to go

File: home.css [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+body::-webkit-scrollbar-thumb {
+  background: var(--primary-color);
+}
+
+header {
+  box-sizing: border-box;
+  background-color: var(--primary-color);
+  display: flex;
+  align-items: center;
+  /* justify-content: space-between; */
```
Comment (kart-c) [inline home.css:21]: remove comment if not necessary

File: home.css [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+  padding: 2rem;
+  box-shadow: 0 3px 5px -1px #0003, 0 6px 10px #00000024, 0 1px 18px #0000001f;
+}
+
+header > h3 {
+  margin: 0 20px;
+}
+.hamburger-menu {
+  display: relative;
```
Comment (kart-c) [inline home.css:30]: this property value is not valid. You might be looking for the `position` property

**Review by kart-c — APPROVED** [review]
a few suggestions I have 
- you can add some padding to the footer element to give some breathing space to the `footer-bottom` container 
- I would suggest to use readable `font-color` on the side navigation. Right now the font color is black and it's background is grey, this makes it difficult to read.

Other than this, the code looks good.

File: home.css (hunk 1 above)
Comment (DevRaahul) [reply home.css:21]: Ok will remove this in next PR.

File: home.css (hunk 2 above)
Comment (DevRaahul) [reply home.css:30]: resolved

**Pull request merged**